src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss

```diff
@@ -118,6 +119,34 @@
     font-weight: 600;
 }
 
+.status {
+    padding-inline: 0.5rem;
+    padding-block: 0.185rem;
+    border-radius: 1rem;
+    font-size: 0.7rem;
+    font-weight: 700;
+    text-transform: capitalize;
+    letter-spacing: 0.04rem;
+    transform: translateY(0.075rem);
+    color: #ffffff;
+
+    &.draftStatus {
+        background: #FFDAA6;
+        color: #1A0F00;
+
+    }
+
+    &.activeStatus {
+        background: #CEF2CF;
+        color: #022603;
+    }
+
+    &.archiveStatus {
+        background: #dadada;
+        color: #0e0e0e;
+    }
+}
+
```

Comment:
If we use the `<StatusBadge />` component as mentioned before, we don't need these styles for status.